src/pytest_mock/plugin.py

```diff
@@ -87,7 +84,7 @@ def spy(self, obj: object, name: str) -> unittest.mock.MagicMock:
 
         :param object obj: An object.
         :param unicode name: A method in object.
-        :rtype: mock.MagicMock
+        :rtype: unittest.mock.MagicMock
```

Comment:
do we care about `:rtype:`s here? can we use sphinx-autodoc-typehints instead?

Reply:
Hey @graingert! 

IMHO we can drop all types from the docs and rely solely on the type annotations.

About sphinx, we don't really have sphinx docs at the moment, so I don't think it matters.

Want to tackle this in this PR?

Reply:
I think worth defering to another pr